dags/direct2parquet_bigquery_load.py

```diff
@@ -62,6 +62,21 @@
             'dataset_version': 'v1',
             'bigquery_dataset': 'telemetry',
             },
+        'coverage-coverage-parquet': {
+            'dataset_version': 'v1',
+            },
+        'firefox-launcher-process-launcher-process-failure-parquet': {
+            'dataset_version': 'v1',
+            },
+        'pocket-fire-tv-events-parquet': {
+            'dataset_version': 'v1',
+            },
+        'webpagetest-webpagetest-run-parquet': {
+            'dataset_version': 'v1',
+            },
+        'telemetry-ip-privacy-parquet': {
```

Comment:
This is a static dataset but I've added it here for convenience. In fact, we could potentially just run this script manually once for all these datasets (we only want historical data), but I don't know have experience doing this.